std/meta.d

```diff
@@ -955,15 +955,18 @@ unittest
 }
 
 
-// : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : //
-package:
-
-/*
- * With the builtin alias declaration, you cannot declare
- * aliases of, for example, literal values. You can alias anything
- * including literal values via this template.
+/**
+ * With the builtin alias declaration, you cannot declare aliases of,
```

Comment:
Nitpick: please don't use personal pronouns in documentation, use passive voice (i.e. "cannot be declared")